creator-node/src/diskManager.ts

```diff
@@ -454,20 +475,23 @@ async function _copyLegacyFiles(
           )
         : computeFilePathAndEnsureItExists(nonLegacyPathInfo.outer))
 
-      // Copy to new path if it doesn't already exist
-      if (!(await fs.pathExists(nonLegacyPath))) {
-        try {
-          await fs.copyFile(legacyPath, nonLegacyPath)
-        } catch (e: any) {
-          // If we see a ENOSPC error, log out the disk space and inode details from the system
-          if (e.message.includes('ENOSPC')) {
-            await Promise.all([
-              runShellCommand(`df`, ['-h'], logger),
-              runShellCommand(`df`, ['-ih'], logger)
-            ])
-          }
-          throw e
+      // Copy to new path, overwriting if it already exists
+      try {
+        // Set the file's mtime to now so disk pruning doesn't delete it before we update db
```

Comment:
what if we just didn't run diskpruning while this is running? is that safer / worth it?

Reply:
to clarify - i meant that in the code, these functions are run one after the other, but that might require more reworking since this loops
if this works, then i think fine to keep

Reply:
oh gotcha. yeah I think I'd worry about disk space growing too much if migrating takes too long. probably not an issue I guess, just not sure with a larger data set

Reply:
agree